parsers/AU.py

```diff
@@ -4,8 +4,18 @@
 import requests
 
 import json
+import numpy as np
 import pandas as pd
-import math
+
+try:
```

Comment:
Hmm this sounds overkill.
Maybe we can add the PYTHONPATH to the pylint command here:
https://github.com/tmrowco/electricitymap/blob/master/circle.yml#L20